README.rst

```diff
@@ -1,5 +1,6 @@
 Python Client for Cloud AutoML API
 ==================================
+**AutoML API Python Client is now available in the new Vertex AI.** Migrate your AutoML resources to Vertex AI to get the latest features like data labeling, AutoML-powered training, and endpoint management. `Learn More`_.
```

Comment:
To stay consistent with the rest of the documentation, consider changing link text to `Migrate to Vertex AI Documentation` on a separate line. It will also improve the experience for users who are checking for links through the screen reader.

Reply:
Addressed in latest commit